AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContextDelegate.h

```diff
@@ -6,13 +6,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol MSAuthTokenContextDelegate <NSObject>
 
+@optional
+
 /**
  * A callback that is called when an auth token is received.
  *
  * @param authTokenContext The auth token context.
  * @param authToken The auth token.
  */
-- (void)authTokenContext:(MSAuthTokenContext *)authTokenContext didReceiveAuthToken:(NSString *)authToken;
+- (void)authTokenContext:(MSAuthTokenContext *)authTokenContext didReceiveAuthToken:(nullable NSString *)authToken;
```

Comment:
I think we might want to rename a bit these callbacks to reflect the action they are triggered for. Can be `didSetNewAuthToken` and `didSetNewAccountIdWithAuthToken`?

Reply:
agree with @clpolet